src/GraphQL/CacheableGraphqlValue.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @file
+ * Wrapper class for values resolved through GraphQL resolvers, which includes
+ * Drupal cache metadata.
+ */
+
+namespace Drupal\graphql\GraphQL;
+
+use Drupal\Core\Cache\CacheableDependencyInterface;
+use Drupal\Core\Cache\CacheableMetadata;
+
+class CacheableGraphqlValue extends CacheableMetadata {
+
+  /**
+   * @var mixed The actual value being wrapped.
```

Comment:
Can you break the description string into its own line like we do in other places conforming to the Drupal coding standards?